js/siteorigin-panels.js

```diff
@@ -3469,13 +3469,23 @@ module.exports = Backbone.View.extend( {
         // Make sure we actually need to copy content.
         if( panelsOptions.copy_content && this.attachedToEditor && this.$el.is(':visible')) {
```

Comment:
Does the Home Page interface attach itself to the editor? Does it pass `this.attachedToEditor`?